Source/Boxed.AspNetCore.TagHelpers/HttpContextExtensions.cs

```diff
@@ -0,0 +1,36 @@
+namespace Boxed.AspNetCore.TagHelpers
+{
+    using System;
+    using Microsoft.AspNetCore.Http;
+    using Microsoft.AspNetCore.Http.Features;
+    using Microsoft.AspNetCore.Mvc;
+    using Microsoft.AspNetCore.Mvc.Infrastructure;
+    using Microsoft.AspNetCore.Mvc.Routing;
+    using Microsoft.Extensions.DependencyInjection;
+
+    /// <summary>
+    /// <see cref="HttpContext"/> extension methods.
+    /// </summary>
+    public static class HttpContextExtensions
+    {
+        /// <summary>
+        /// Gets UrlHelper instance using <see cref="IUrlHelperFactory"/> and <see cref="IActionContextAccessor"/>.
+        /// </summary>
+        /// <param name="httpContext">The HTTP context.</param>
+        /// <returns><see cref="UrlHelper"/> instance for current request.</returns>
+        public static IUrlHelper GetUrlHelper(this HttpContext httpContext)
```

Comment:
Good idea to make this an extension method. I think it would make better sense in the Boxed.AspNetCore package though. Also the code can be simplified:

```
/// <summary>
/// Gets an <see cref="IUrlHelper"/> instance from the <paramref name="httpContext"/>.
/// </summary>
/// <param name="httpContext">The HTTP context.</param>
/// <returns>An instance of the <see cref="IUrlHelper"/>.</returns>
public static IUrlHelper GetUrlHelper(this HttpContext httpContext)
{
    var services = httpContext.RequestServices;
    var actionContext = services
        .GetRequiredService<IActionContextAccessor>()
        .ActionContext;
    var urlHelper = services
        .GetRequiredService<IUrlHelperFactory>()
        .GetUrlHelper(actionContext);
    return urlHelper;
}
```